src/hevm/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # hevm changelog
 
+## Unreleased
+
+### Added
+
+- Two new cheatcodes were added: `sign(uint sk, bytes message)` and `addr(uint sk)`. Taken together
```

Comment:
might be good to mention the bump in nixpkgs / ghc too (also in `dapp`, `ethsign` and `seth`)